### PR TITLE
feat(learn): python certification projects

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

**Review by scissorsneedfoodtoo — COMMENTED** [review]
This is looking great @beaucarnes. Will be exciting to get these live.

I noticed a some things that I left as review comments. Also, there are some other things I found and am still investigating:
* Can't claim "Scientific Computing in Python" certificate for some reason
* Was able to claim the Machine Learning and Data Analysis certificates, but can't see the certificate after refreshing the page (the "Claim Certificate" button doesn't change to "Show Certification" for some reason)
* Do we need to set up forum posts for these beforehand and include the id in the frontmatter (`forumTopicId`)?
* Are we splitting up the Information Security and Quality Assurance section with this PR? A couple of my projects aren't ready yet, and we would need to set up a legacy certificate for the original 5 projects. Would we be able to hold off on this until the other projects are ready and we have the legacy projects/certificate set up? Otherwise I think the IS&QA cert will require 7 projects.

**Comment by scissorsneedfoodtoo** [comment]
It seems like I can't push directly to this branch while this is in draft mode. I have a commit ready that will fix some of the minor typos and change the id for the Sea Level Predictor project.

**Comment by scissorsneedfoodtoo** [comment]
Thanks for marking this ready for review @raisedadead.

Tried pushing my changes to the branch but am still getting a permission denied error. Maybe there's some branch protection? Either way, I'll keep testing this locally.

File: api-server/server/boot/certificate.js [hunk 1]
```diff
@@ -163,7 +181,10 @@ function sendCertifiedEmail(
     isJsAlgoDataStructCert,
     isDataVisCert,
     isApisMicroservicesCert,
-    isInfosecQaCert
+    isInfosecQaCert,
+    isSciPyCert,
+    isDataAnalyisisPyCert,
```
Comment (scissorsneedfoodtoo) [inline api-server/server/boot/certificate.js]: isDataAnalysisPyCert,

**Comment by raisedadead** [comment]
Oh, you are right. I totally forgot. It's Beau's fork and I am the OP for the PR. You should be able to create code review suggestions and Beau can apply them manually on his local and push the changes up.

File: api-server/server/boot/certificate.js [hunk 2]
```diff
@@ -174,7 +195,10 @@ function sendCertifiedEmail(
     !isJsAlgoDataStructCert ||
     !isDataVisCert ||
     !isApisMicroservicesCert ||
-    !isInfosecQaCert
+    !isInfosecQaCert ||
+    !isSciPyCert ||
+    !isDataAnalyisisPyCert ||
```
Comment (scissorsneedfoodtoo) [inline api-server/server/boot/certificate.js]: !isDataAnalysisPyCert ||

File: api-server/server/boot/certificate.js [hunk 3]
```diff
@@ -339,6 +369,9 @@ function createShowCert(app) {
       is2018DataVisCert: true,
       isApisMicroservicesCert: true,
       isInfosecQaCert: true,
+      isSciPyCert: true,
+      isDataAnalyisisPyCert: true,
```
Comment (scissorsneedfoodtoo) [inline api-server/server/boot/certificate.js]: isDataAnalysisPyCert: true,

**Comment by scissorsneedfoodtoo** [comment]
@raisedadead, oh I see. That makes sense. Sure, I'll keep working on this and leave review comments.

**Comment by beaucarnes** [comment]
@scissorsneedfoodtoo Thanks for the review. I fixed the errors you found.

I'll look into the issue about claiming certificates. That is something I forgot to check before.

As for splitting up the Information Security and Quality Assurance section, I figured we would just have a 7 project IS&QA certificate until the rest of the QA JavaScript projects are finished. Then we would split the certificates at that time. We should confirm with @QuincyLarson about what he thinks.

@QuincyLarson Can you create forum sections for the new Python projects.

**Comment by scissorsneedfoodtoo** [comment]
@beaucarnes, okay, that sounds good. Will wait to hear back from @QuincyLarson about that certificate. In the meantime I'll double check all of the links to the projects and also look into the other certificates to see if we're missing something, maybe a Redux action.

**Comment by raisedadead** [comment]
@RandellDawson would you be interested in guiding us on the templates for the forum topics for these new projects, thanks.

**Comment by raisedadead** [comment]
I created a branch and landed it there instead of the master branch. We will land this on .dev, but Randell is going to make a PR that updates the forum topic ids into the same.

**Comment by RandellDawson** [comment]
@raisedadead Just to confirm, since the projects do not have solutions, there will be no need to create hints/solutions topics for these?

**Comment by raisedadead** [comment]
Correct, I will cleanup the forumTopicId from any of the challenges on here.

**Pull request merged**